### PR TITLE
Configurabel products special price bug 10303

### PR DESCRIPTION
**Pull request by nadeefit6**
### Description (*)
vendor/magento/module-swatches/view/frontend/web/js/swatch-renderer.js
-            selectorProduct: '.product-info-main',
+            selectorProduct: '.--product-info-main',

### Fixed Issues (if relevant)
1. magento/magento2#10303: Configurabel products special price bug
2. ...

### Manual testing scenarios (*)

1. Create configurable product with simple products
2. Add special Prices to Simple products associated with the configurable product
3. Clear Cache
4. Go to Product detail page
5. Change the configurable option drop-down(Ex:size) and check Price
6. Special Price of simple product should be displayed

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @nadeefit6. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me 2.3-develop instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/code/Magento/Paypal/Controller/Express/AbstractExpress/PlaceOrder.php [hunk 1]
```diff
@@ -110,8 +110,7 @@ public function execute()
                 $this->_getCheckoutSession()->setLastOrderId($order->getId())
                     ->setLastRealOrderId($order->getIncrementId())
                     ->setLastOrderStatus($order->getStatus());
-            }
-
+            }
```
Comment (orlangur) [inline app/code/Magento/Paypal/Controller/Express/AbstractExpress/PlaceOrder.php:113]: Revert changes in this file.

File: app/code/Magento/Swatches/view/frontend/web/js/swatch-renderer.js [hunk 2]
```diff
@@ -217,7 +217,7 @@ define([
             jsonSwatchConfig: {},
 
             // selector of parental block of prices and swatches (need to know where to seek for price block)
-            selectorProduct: '.product-info-main',
+            selectorProduct: '.--product-info-main',
```
Comment (orlangur) [inline app/code/Magento/Swatches/view/frontend/web/js/swatch-renderer.js:220]: Such selector is actually incorrect and violates https://devdocs.magento.com/guides/v2.3/coding-standards/code-standard-demarcation.html

Please add appropriate data attributes instead and add here.

**Review by orlangur — CHANGES_REQUESTED** [review]
Thanks @nadeefit6 for your contribution! Let's wait for issue to be confirmed and then apply necessary changes to PR please.

**Comment by sidolov** [comment]
@nadeefit6 , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**